.github/workflows/deploy.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           BASIC_AUTH_USER: ${{ secrets.BASIC_AUTH_USER }}
           BASIC_AUTH_PASS: ${{ secrets.BASIC_AUTH_PASS }}
           SUPPORT_EMAIL: ${{ secrets.SUPPORT_EMAIL }}
+          MODSEC_CONFIG: ${{ inputs.modsec_config }}
```

Comment:
Will this expose us?

Reply:
... or the security rules?? 😄

Reply:
Good question, I'm 99.9% sure that read access to the repo doesn't allow inspecting the steps of workflows.

btw, it's the same way we pass the IPs arround in the intranet repo, I think I checked then.

I can login with my alt GH account and double check.

Reply:
Hey @EarthlingDavey, I hope you had a great weekend mate, let's look at this today.